### PR TITLE
HS-1110: Rules and completed policies cards

### PR DESCRIPTION
**Pull request by mikewrosey**
## Description
Changes to rules form and conditions. Starts the cards/list of cards for completed policies & rules

screenshot-localhost_3009-2023 03 17-16_20_01

screenshot-localhost_3009-2023 03 17-16_17_40

## Jira link(s)
- https://issues.opennms.org/browse/HS-1110
- https://issues.opennms.org/browse/HS-1111

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesCard.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+</script>
+
+<style scoped lang="scss">
+@use '@featherds/styles/themes/variables';
+@use '@featherds/styles/mixins/typography';
+@use '@featherds/styles/mixins/elevation';
+@use '@/styles/mediaQueriesMixins.scss';
+.mp-card {
+  @include elevation.elevation(3);
+  border-radius: 5px;
```
Comment (chinh-cle) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesCard.vue:71]: we may use `$border-radius-s` in vars.scss to have consistency across screens. Particularly the `border-radius: 10px;` in MonitoringPoliciesExistingItems.vue

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesCard.vue [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+      cursor: pointer;
+      color: var(variables.$primary);
+      text-decoration: underline;
+    }
+
+    .title-box {
+      white-space: nowrap;
+      background: var(variables.$shade-4);
+      padding: var(variables.$spacing-xs);
+      border-radius: 3px;
```
Comment (chinh-cle) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesCard.vue:101]: we may use `$border-radius-xs` in vars.scss to have consistency across screens.

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesCard.vue [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        flex: 2;
+      }
+      &.component {
+        flex: 1;
+      }
+    }
+
+    .alert-conditions-btn {
+      @include typography.subtitle2;
+      border-radius: 3px;
```
Comment (chinh-cle) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesCard.vue:122]: we may use `$border-radius-xs` in vars.scss to have consistency across screens.

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesCardAlertRow.vue [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  .alert-letter {
+    @include typography.subtitle1;
+    display: flex;
+    align-items: center;
+    height: 30px;
+    width: 30px;
+    background: var(variables.$primary);
+    padding: var(variables.$spacing-xs);
+    color: var(variables.$primary-text-on-color);
+    border-radius: 50%;
```
Comment (chinh-cle) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesCardAlertRow.vue:69]: altho this one is pretty obvious, we may want to add it to vars.scss to something like `$border-radius-round/circle`.

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesThresholdCondition.vue [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+.condition {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  flex-direction: column;
+  gap: var(variables.$spacing-xs);
+  padding: var(variables.$spacing-xl) var(variables.$spacing-s);
+  margin-top: var(variables.$spacing-s);
+  border: 1px solid var(variables.$shade-4);
+  border-radius: 5px;
```
Comment (chinh-cle) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesThresholdCondition.vue:127]: we may use $border-radius-s in vars.scss to have consistency across screens.

File: ui/src/store/Views/monitoringPoliciesStore.ts [hunk 6]
```diff
@@ -22,20 +25,38 @@ const defaultPolicy: IPolicy = {
...
 export const useMonitoringPoliciesStore = defineStore('monitoringPoliciesStore', {
   state: (): TState => ({
     selectedPolicy: undefined,
     selectedRule: undefined
   }),
+  getters: {
+    monitoringPolicies() {
+      const { monitoringPolicies } = useMonitoringPoliciesQueries()
+      return monitoringPolicies
+    }
```
Comment (chinh-cle) [inline ui/src/store/Views/monitoringPoliciesStore.ts:58]: ok I see what you were talking about.

**Review by chinh-cle — APPROVED** [review]
few minor things but LGTM

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesCardAlertRow.vue [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+    <div class="col box double">{{ condition.periodUnit }}</div>
+    <div class="col half">as</div>
+    <div class="col box double">{{ condition.severity }}</div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ICondition } from '@/types/policies'
+
+const conditionLetters = ['a.', 'b.', 'c.', 'd.']
```
[the remark this answers is not in the document]
Comment (mikewrosey) [reply ui/src/components/MonitoringPolicies/MonitoringPoliciesCardAlertRow.vue:21]: I think it's just one for each severity, but I'll double check. This list might expand.

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesThresholdCondition.vue (hunk 5 above)
Comment (mikewrosey) [reply ui/src/components/MonitoringPolicies/MonitoringPoliciesThresholdCondition.vue]: done!

**Pull request merged**